frontend/src/components/homepage/helpers/convert-to-podcast-type.ts

```diff
@@ -0,0 +1,9 @@
+
+import { FieldValues, UnpackNestedValue } from 'react-hook-form';
+import { PodcastBySearchPayload } from 'common/types/types';
+
+const convertToPodcastType = (pref: UnpackNestedValue<FieldValues>): PodcastBySearchPayload => {
```

Comment:
we don't need it

Reply:
can't delete coz it use to convert to type, that action accept